### PR TITLE
Incorrect error message bugfix

### LINKED ISSUE #147: [PE-D][Tester B] Duplicate person issue
<i><video controls><source src="https://raw.githubusercontent.com/Potatoad88/ped/main/files/a89676b0-efac-42a8-9aba-fe67a9cedc1a.mov" type="video/mp4">Your browser does not support the video tag.</video><br>video:https://raw.githubusercontent.com/Potatoad88/ped/main/files/a89676b0-efac-42a8-9aba-fe67a9cedc1a.mov</i>
I tried adding a new person but I got the error saying that this person already exists although even after I call the list command I don't see this person in my addressbook.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: Potatoad88/ped#5

### PR DESCRIPTION
**Pull request by volleyballkickedme**
Attempting to add a person with a pre existing NRIC displays a corresponding error message

close #147

### CODE REVIEW

**Review by reidenong — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -42,12 +42,15 @@ public class AddCommand extends Command {
...
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TRIAGE + "1"
-            + PREFIX_TAG + "Pacemaker "
-            + PREFIX_TAG + "High Blood Pressure";
+            + PREFIX_TRIAGE + "1 "
+            + PREFIX_TAG + "Pacemaker";
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_NRIC = """
```
Comment (itsnotsherm) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:50]: good clarity of message

**Review by itsnotsherm — APPROVED** [review]
LGTM

**Review by Dominic-Khoo — APPROVED** [review]
LGTM

**Pull request merged**